config/jobs/kubernetes-sigs/metrics-server/metrics-server-presubmits.yaml

```diff
@@ -0,0 +1,87 @@
+presubmits:
+  kubernetes-sigs/metrics-server:
+  - name: pull-metrics-server-verify
+    decorate: true
+    path_alias: sigs.k8s.io/metrics-server
+    always_run: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200312-a0f211f-master
+        command:
+        - runner.sh
+        - make
+        - lint
+    annotations:
+      testgrid-dashboards: sig-instrumentation-metrics-server
+      testgrid-tab-name: pr-verify
+  - name: pull-metrics-server-test-unit
+    decorate: true
+    path_alias: sigs.k8s.io/metrics-server
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200312-a0f211f-master
+        args:
```

Comment:
here should be `command`, like other jobs above.

Reply:
done